src/config/national.js

```diff
@@ -48,14 +62,32 @@ export const NationalConfig = {
       title: 'MAP_SETTLEMENTS_TITLE',
       zIndex: 103
     }],
+    // The Leaflet map options, should be of the form:
```

Comment:
To me this feels like it's saying it should be of the form of a URL, maybe say "should be of the form described here:"?